notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>A more complete explanation of INITIAL_GH_CUT_EFFICENCY is needed</p><p>Number of E bins is given, but not range, better if they are both here</p><p>Typo: INSENSITY</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done!</p><p>This initial INITIAL_GH_CUT_EFFICENCY is used to calculate initial theta2 cuts with a fixed g/h cut (which corresponds to that efficiency), then the g/h cut is optimized after applying these initial theta2 cuts. The final theta2 cuts are calculated after the g/h cuts optimization. I followed the procedure in pyirf/examples/calculate_eventdisplay_irfs.py for this.</p>